Teapot/Teapot.swift

```diff
@@ -18,21 +18,15 @@ open class Teapot {
         DispatchQueue(label: "com.backkenbaeck.Teapot.RunTaskQueue", qos: .background)
     }()
 
-    open lazy var queue: OperationQueue = {
-        let queue = OperationQueue()
-        queue.name = "no.bakkenbaeck.NetworkQueue"
-        queue.qualityOfService = .userInitiated
+    open var queue: OperationQueue
```

Comment:
why does this need to be a `var` instead of just a `let`?

Reply:
Because otherwise it can't be `open`. `let`s are implicitly `final`.